demo/package.json

```diff
@@ -2,61 +2,41 @@
   "nativescript": {
     "id": "org.nativescript.demo",
     "tns-android": {
-      "version": "5.3.1"
+      "version": "6.5.0"
     },
     "tns-ios": {
-      "version": "5.4.0"
+      "version": "6.5.0"
     }
   },
   "description": "A NativeScript demo app for Live Engage",
+  "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/Essent/nativescript-live-engage.git"
   },
-  "license": "MIT",
   "dependencies": {
-    "@angular/common": "~6.1.0",
-    "@angular/compiler": "~6.1.0",
-    "@angular/core": "~6.1.0",
-    "@angular/forms": "~6.1.0",
-    "@angular/http": "~6.1.0",
-    "@angular/platform-browser": "~6.1.0",
-    "@angular/platform-browser-dynamic": "~6.1.0",
-    "@angular/router": "~6.1.0",
-    "nativescript-angular": "~6.2.0",
+    "@angular/animations": "~8.2.0",
+    "@angular/common": "~8.2.0",
+    "@angular/compiler": "~8.2.0",
+    "@angular/core": "~8.2.0",
+    "@angular/forms": "~8.2.0",
+    "@angular/platform-browser": "~8.2.0",
+    "@angular/platform-browser-dynamic": "~8.2.0",
+    "@angular/router": "~8.2.0",
+    "@nativescript/theme": "~2.3.3",
+    "nativescript-angular": "~8.21.0",
     "nativescript-live-engage": "file:../src",
-    "nativescript-theme-core": "^1.0.4",
-    "nativescript-unit-test-runner": "^0.3.4",
-    "rxjs": "^6.3.3",
-    "tns-core-modules": "^5.4.0"
+    "reflect-metadata": "~0.1.12",
+    "rxjs": "^6.4.0",
+    "tns-core-modules": "~6.5.0",
+    "zone.js": "~0.9.1"
   },
   "devDependencies": {
-    "@angular/compiler-cli": "~6.1.0",
-    "@ngtools/webpack": "~6.2.0",
-    "babel-traverse": "6.12.0",
-    "babel-types": "6.26.0",
-    "babylon": "6.18.0",
-    "copy-webpack-plugin": "~4.0.1",
-    "extract-text-webpack-plugin": "~3.0.0",
-    "filewalker": "0.1.3",
```

Comment:
We don't need all of this deleted dependencies?

Reply:
It works without them.  :)